### DISCUSSION TITLE
How to pass Docker container config dynamically in DockerRunLauncher ?

### DISCUSSION THREAD

**Discussion by dmarteau** [dagster-io/dagster#14540] [Q&A]
Hi,

Is there a way to pass container configuration  to DockerRunLauncher as job run config ?

Example: Passing different bind mount dynamically to job's run.

Thanks.

**Comment by clairelin135** [comment]
Hi @dmarteau. I believe it's currently only possible to pass config to `DockerRunLauncher` via the `dagster.yaml` file.
